### PR TITLE
Footer Mapping

### LINKED ISSUE #96: Map Through Social Media Icons in Footer
- Currently the footer icons is hard coded and messy/long
- We want to shorten the code such that we only have 1 Row/Col
- You will need to create a const array (do not need a useState) and map through that data and populate the topic and color fields of the Field appropriately
- This work will be done solely inside the Footer.js file

### PR DESCRIPTION
**Pull request by Sarah-Rami27**
closes #96

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -9,6 +9,53 @@ import {
...
+		image: <FaSlack size='56'></FaSlack>,
+	},
+	{
+		path: "https://medium.com/acm-at-ucr",
+		title: "Medium",
+		image: <FaMediumM size='56'></FaMediumM>,
+	},
+];
+
+const FooterLink = ({ path, title, image }) => {
```
Comment (shahdivyank) [inline components/Footer.js]: Can you integrate this directly into the footer?

File: components/Footer.js [hunk 2]
```diff
@@ -9,6 +9,53 @@ import {
...
+	return (
+		<a
```
Comment (shahdivyank) [inline components/Footer.js]: use `<Link>` tags

File: components/Footer.js [hunk 3]
```diff
@@ -9,6 +9,53 @@ import {
...
+			rel='noopener noreferrer'
+			href={path}
+			target='_blank'
+			title={title}
+			className='mx-auto fill-current md:mx-12 lg:m-0 text-gray-50 hover:text-gray-300'
```
Comment (shahdivyank) [inline components/Footer.js]: remove the margins, Bootstrap Grid should handle the spacing

Comment (shahdivyank) [inline components/Footer.js]: Let's use the custom tailwind acm colors

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**